src/main/java/com/aetherteam/aether_genesis/item/accessories/CrystalBottleItem.java

```diff
@@ -0,0 +1,58 @@
+package com.aetherteam.aether_genesis.item.accessories;
+
+import com.aetherteam.aether.capability.player.AetherPlayer;
+import com.aetherteam.aether.item.accessories.AccessoryItem;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResultHolder;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.TooltipFlag;
+import net.minecraft.world.level.Level;
+import top.theillusivec4.curios.api.SlotContext;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+public class CrystalBottleItem extends AccessoryItem {
+    public CrystalBottleItem(Properties properties) {
+        super(properties);
+    }
+
+    public InteractionResultHolder<ItemStack> use(Level level, Player player, InteractionHand hand)  {
+        ItemStack heldItem = player.getItemInHand(hand);
+        AetherPlayer.get(player).ifPresent(aetherPlayer -> {
```

Comment:
- Usage of the AetherPlayer capability is probably not necessary here, you should just be able to use the `player` parameter in use.
- The experience tooltip doesn't seem to work properly, although it didn't work in Genesis of the Void either, but it should probably be fixed here.
- The use should only return a success if it is emptying or filling xp, otherwise it should be a pass by default.
- The accessory should be allowed to be equipped from using but only depending on if the player isn't performing any xp filling or emptying.
-

Reply:
okay, sorry for my bad coding...